cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-12-16 12:51
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(
+                null=True, upload_to=cinema.models.movie_image_path

```

Comment:
Ensure that `cinema.models.movie_image_path` is defined in the `cinema.models` module. This function should generate the correct file path for the uploaded images, following the format specified in the task description: `f"{slugify(movie.title)}-{uuid}{ext}"`.